README.md

```diff
@@ -324,7 +325,7 @@ For the full list of arguments available, see the documentation in [docs/cli-arg
 
 #### Helm Chart
 
-The official Helm chart is maintained in [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics).
+The official [Helm chart](https://artifacthub.io/packages/helm/prometheus-community/kube-state-metrics/) is maintained in [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics).
```

Comment:
Starting from `kube-state-metrics v2.13.3`, the official Helm chart is maintained at [prometheus-community/helm-charts](https://github.com/prometheus-community/helm-charts/tree/main/charts/kube-state-metrics).